src/main/java/com/aetherteam/aether_genesis/capability/player/GenesisPlayerCapability.java

```diff
@@ -0,0 +1,39 @@
+package com.aetherteam.aether_genesis.capability.player;
+
+import com.aetherteam.aether.capability.player.AetherPlayerCapability;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.world.entity.player.Player;
+
+public class GenesisPlayerCapability extends AetherPlayerCapability implements GenesisPlayer {
```

Comment:
You shouldn't have to extend AetherPlayerCapability for this, just CapabilitySyncing

Reply:
got it